zulipterminal/model.py

```diff
@@ -145,7 +148,7 @@ def __init__(self, controller: Any) -> None:
         for name, data in unicode_emoji_data.items():
             data['type'] = 'unicode_emoji'
         typed_unicode_emoji_data = cast(NamedEmojiData, unicode_emoji_data)
-        custom_emoji_data = self.fetch_custom_emojis()
+        custom_emoji_data = self.initial_data['realm_emoji']
```

Comment:
This sets `custom_emoji_data` to a different data format than before; note the processing done in `fetch_custom_emojis` - we could rename and adjust that to process the realm emoji in the initial data instead.

In principle we could adjust the code to handle original and emoji events to be in the same format, but we still need to integrate those with the format of the `active_emoji_data` upon update.